jjb/duffy/testing/sig-paas.yml

```diff
@@ -0,0 +1,73 @@
+#- job:
```

Comment:
Possibly, but as I said above, not at the moment. I want to ensure I work with @cgwalters to get his bits in here. When I do, this stuff may become relevant.